serializers/web/github_runner.rb

```diff
@@ -22,6 +22,8 @@ def self.base(runner)
                   runner.vm.display_state
                 elsif runner.strand.label == "wait_vm_destroy"
                   "deleted"
+                elsif runner.strand.label == "waiting_for_customer_concurrency_limit"
+                  "customer_concurrency_limit_reached"
```

Comment:
Can you shorten it a little bit? Probably, the very long will blow the table on UI